lessons/advanced/otp-supervisors.md

```diff
@@ -42,6 +42,16 @@ There are currently four different restart strategies available to supervisors:
 
 + `:simple_one_for_one` - Best for dynamically attached children. Supervisor spec is required to contain only one child, but this child can be spawned multiple times. This strategy is intended to be used when you need to dynamically start and stop supervised children.
 
+### Restart values
+
+There are several approaches for handling child process crashes:
+
+* `:permanent` - Child is always restarted.
```

Comment:
Worth mentioning that the definitions from the strategies list (several lines above) start with a capital letter.